spider.js

```diff
@@ -73,6 +81,11 @@
 
     // Add the URL to visited stack
     visitedUrls.push(url);
+    
+    // Add cookie
+    if (dataObj.cookie) {
```

Comment:
`undefined` is false-y: http://james.padolsey.com/javascript/truthy-falsey/

Is there a chance this will find its way into the master branch?